vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -21,9 +21,15 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"strings"
 	"time"
 
 	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/informers"
+	kube_client "k8s.io/client-go/kubernetes"
+	kube_flag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog/v2"
+
```

Comment:
I don't understand why they don't force import ordering upstream.
Maybe something we should propose if it is not already applied on their `master`